third_party/xxhash/tests/Makefile

```diff
@@ -0,0 +1,132 @@
+// clang-format off
```

Comment:
TODO: do we want to support tests in third_party?

Reply:
Generally speaking, yes. It'd be very much welcome if you went to the extra effort to do that.